Atomic/pull.py

```diff
@@ -27,6 +29,9 @@ def pull_docker_image(self):
         tag = tag if tag != "" else "latest"
         fq_name = skopeo_inspect("docker://{}".format(self.args.image))['Name']
         image = "docker-daemon:{}:{}".format(fq_name, tag)
+        if get_atomic_config_item(['discover_sigstores'], get_atomic_config()):
+            if not self.discover_sigstore():
+                write_out("There was a problem configuring the trust policy")
```

Comment:
The user is pulling an image. I don't want to exit if we cannot automagically configure trust but I do want to warn them and provide further steps to manually configure trust.